app/views/account/you-are-signed-in.html

```diff
@@ -0,0 +1,20 @@
+
+{% extends "_templates/form-template.html" %}
+
+{% set pageHeading = "Do you want to search for a thing?" %}
+{% set backLink = "false" %}
+{% set formAction = "/application/start" %}
+{% set signedOut = "true" %}
+
+{% block formContent %}
+
+  <h1 class="govuk-heading-xl">You are signed in to GOV.UK</h1>
+
+  <p class="govuk-body">We will now ask you a few questions about your work experience and qualifications.</p>
```

Comment:
Suggested copy update below! 

We probably also need to consider a scenario here where they have already finished an application and have bene accepted as an expert. In that instance the content might be something like  "Continue to your access you account". 

```suggestion
  <p class="govuk-body">Continue to apply to become to be a subject matter expert.</p>
```